src/ontology/cl-edit.owl

```diff
@@ -31685,6 +31687,31 @@ AnnotationAssertion(rdfs:label obo:CL_4033056 "onychocyte")
 SubClassOf(obo:CL_4033056 obo:CL_0000312)
 SubClassOf(obo:CL_4033056 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0008198))
 
+# Class: obo:CL_4033059 (lactocyte type 1)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35091553") Annotation(oboInOwl:hasDbXref "PMID:35377806") obo:IAO_0000115 obo:CL_4033059 "A lactocyte that highly expresses genes associated with transcription, immune cell function, and cellular stress. A lactocyte type 1 also expresses genes involved in milk component biosynthesis (e.g., LALBA and CSNs), albeit at lower levels than a lactocyte type 2.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033059 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033059 <https://orcid.org/0000-0003-4361-817X>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033059 "2023-10-16T13:15:36Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35377806") oboInOwl:hasBroadSynonym obo:CL_4033059 "luminal cluster 1 cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35091553") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033059 "LC1")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35091553") oboInOwl:hasRelatedSynonym obo:CL_4033059 "LMC major luminal cluster 1")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35091553") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033059 "progenitor MEC")
+AnnotationAssertion(rdfs:label obo:CL_4033059 "lactocyte type 1")
+SubClassOf(obo:CL_4033059 obo:CL_0002325)
+
+# Class: obo:CL_4033060 (lactocyte type 2)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35091553") Annotation(oboInOwl:hasDbXref "PMID:35377806") obo:IAO_0000115 obo:CL_4033060 "A lactocyte that highly expresses genes associated with lipid production and milk component biosynthesis, suggesting that this cell type is more mature than a lactocyte type 1.")
```

Comment:
- Same as above: Text definition uses a genus that is not otherwise defined in the ontology.

- RE: "...  **suggesting** that this cell type is more mature than a lactocyte type 1", consider moving this to a comment. 

From the editing guidelines:

DO NOT include reasons for believing the assertions to be true. These should be recorded in comments.
DO NOT raise questions in the definition. The definition should have the sense of being definitive, which is undermined if we show doubt. Any doubt should be recorded in comments instead.

Reply:
I have remove the last part of the sentence.